oe_content.post_update.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+/**
+ * @file
+ * OpenEuropa Content post updates.
+ */
+
+declare(strict_types = 1);
+
+/**
+ * Set a max length for title fields on corporate content types.
+ */
+function oe_content_post_update_set_title_max_length(array &$sandbox): void {
+  $target_bundles = [
+    'oe_news',
+    'oe_page',
+    'oe_policy',
+    'oe_publication',
+  ];
+  $available_bundles = \Drupal::service('entity_type.bundle.info')->getBundleInfo('node');
```

Comment:
`\Drupal::entityTypeManager()->getStorage('node_type')->loadMultiple($target_bundles)` will return all the bundle config entities you want. You can loop through those and no need to check anything anymore no?.